samples/README.md

```diff
@@ -0,0 +1,18 @@
+# Samples :books:
+
+We have prepared some sample code for you to try out. Some is written in JavaScript and some in TypeScript.
+
+## JavaScript samples
+
+- [hello](hello) - Sequence that modifies incoming stream of strings by saying Hello :).
+- [hello-snowman](hello-snowman) - Sequence that reads incoming stream, and and modifies it by adding a text message according to the incoming data.
+- [simple-counter-js](simple-counter-js) - Sequence, that counts to 1000, and logs the number in one-second intervals.
+- [test-output](test-output) - Sequence that just writes random values to output stream
```

Comment:
```suggestion
- [test-output](test-output) - Sequence that just writes random values to the output stream
```

Reply:
I know I'm picking on, but should be a period at the end of a sentence 🙈. I love the sample README style I'm smiling at the computer 🤩🌈

Reply:
It should be and it will be. You are not picky 😉  that is why I need two more pairs of eyes 👀 👀  to find all the mistakes 😄 thanks Ada!